package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "backend",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "type": "module",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
```

Comment:
remove test script,
change start to `node index.js` and add dev script to `nodemon index.js`